uwebsockets/protocol.py

```diff
@@ -38,7 +38,8 @@ def urlparse(uri):
     """Parse ws:// URLs"""
     match = URL_RE.match(uri)
     if match:
-        return URI(match.group(1), int(match.group(2)), match.group(3))
+        port = int(80 if match.group(2)==None else match.group(2))
```

Comment:
Let's break this down into something more readable, like
```
host, port, path = match.groups
if port is None:
    port = 80

assert host
assert port
assert path
```

Reply:
Sorry I'm writing from memory, I don't specifically remember how this works in `ure`. I think it might be `_, host, port, path = match.group()` ? Or something like that.

Reply:
Also maybe `path` can be falsey. That might be okay.

Reply:
https://github.com/Hermann-SW/uwebsockets/commit/86f30c654853c109a9c9b9f258fecba6c9b70892